### ISSUE TITLE
About the code "data_aug = torch.cat([inputs_max, data])"

### ISSUE THREAD

**Issue by bregmangh** [BUserName/Learning_to_diversify#2]
Hi, thanks for your help about my issues. But, I have an another question in train.py:
data_aug = torch.cat([inputs_max, data])
should be：
data_aug = torch.cat([data, inputs_max])

Because it has to be consistent with the meaning expressed by the following codes：
emb_src = F.normalize(tuple['Embedding'][:class_l.size(0)]).unsqueeze(1)  
emb_aug = F.normalize(tuple['Embedding'][class_l.size(0):]).unsqueeze(1)
and
mu = tuple['mu'][class_l.size(0):]
logvar = tuple['logvar'][class_l.size(0):]
y_samples = tuple['Embedding'][:class_l.size(0)] 
likeli = -loglikeli(mu, logvar, y_samples)
is it? Thanks.
